Dockerfile

```diff
@@ -2,12 +2,15 @@ FROM kalilinux/kali-rolling
 
 RUN apt-get update && \ 
     apt-get -y upgrade && \
-    apt-get -y install wget kmod procps
+    apt-get -y install wget kmod procps sudo build-essential liblzma-dev liblzo2-dev zlib1g-dev git
 
 WORKDIR /app
 ADD . /app
 
 RUN yes | ./installer.sh
 
-ENTRYPOINT [ "/bin/bash" ]
+RUN cd ./external/cve-search/ && \
+    pip3 install -r requirements.txt && \
+    xargs sudo apt-get install -y < requirements.system
```

Comment:
Would this better placed in the emba installer? Currently our installer is not able to handle cve-search completely. With this it would be on the right track and docker should be also fine.